lib/ansible/plugins/connection/network_cli.py

```diff
@@ -288,6 +288,8 @@ def _connect(self):
         p.force_persistence = self.force_persistence
         ssh = p._connect()
 
+        self.p = p
```

Comment:
`p` can be removed completely and `self.p` can be renamed to a meaningful name something like`self.paramiko_conn`